modules/command-engine/core/src/test/scala/surge/internal/domain/SurgeMessagePipelineSpec.scala

```diff
@@ -295,14 +296,17 @@ class SurgeMessagePipelineSpec
 
       override def actorSystem: ActorSystem = system
 
+      private val isAkkaClusterEnabled = Try(config.getBoolean("surge.akka.cluster.enabled")).getOrElse(false)
```

Comment:
The `Try` here seems overprotective - we should define the setting in our reference.conf so it should always have a value